src/components/AnnualReport/ARDate.vue

```diff
@@ -9,20 +9,12 @@
   </v-card>
 </template>
 
-<script lang="ts">
-// Libraries
-import { Component, Mixins } from 'vue-property-decorator'
-import { State, Getter } from 'vuex-class'
+<script setup lang="ts">
+import { DateComposable, AllowableActionsComposable, FilingComposable } from '@/composables'
 
-// Mixins
-import { DateMixin } from '@/mixins'
-
-@Component({})
-export default class ArDate extends Mixins(DateMixin) {
-  @State nextARDate!: string
-  @Getter isBComp!: boolean
-  @Getter getCurrentDate!: string
-}
+const { formatYyyyMmDd } = DateComposable()
+const { isBComp } = AllowableActionsComposable() // probably can pull from state getters
+const { getCurrentDate } = FilingComposable()
```

Comment:
Should  come from Vuex getters.

Reply:
Yes it does, I only used the getCurrentDate from FilingComposable because I hadn't implemented the getCurrentDate in the Vuex getters and I wanted a complete component

Reply:
Ok.

It might be good to be concise moving forward.

Reply:
Yes I agree